test/unit/config/dom.js

```diff
@@ -22,6 +23,13 @@ chai.use(function(_chai, utils) {
 
 
 describe('DomApi', function() {
+  describe('#setDomApi', function() {
+    it('should return the current class', function() {
+      const MyObject = Backbone.Marionette.Object.extend({}, {setDomApi: setDomApi});
```

Comment:
it'd be nice to do unit tests without having to do a build in the future.. so it'd be nice not to use the global `Backbone` or `Marionette`.  In this case though I think you could just 
```js
const MyObject = function() { };
MyObject.setDomApi = setDomApi;
```